document/src/test/java/com/ritense/document/web/rest/JsonSchemaDocumentResourceTest.java

```diff
@@ -103,7 +106,27 @@ public void shouldReturnOkWithDocument() throws Exception {
     }
 
     @Test
-    public void shouldModifyDocument() throws Exception {
+    void shouldReturnDocumentWithAssignee() throws Exception {
```

Comment:
@marijnritense The test proves that Valtimo already has the required functionality